src/scripts/encodingsettings.js

```diff
@@ -4,7 +4,7 @@ define(["jQuery", "loading", "globalize", "dom"], function($, loading, globalize
     function loadPage(page, config, systemInfo) {
         Array.prototype.forEach.call(page.querySelectorAll(".chkDecodeCodec"), function(c) {
             c.checked = -1 !== (config.HardwareDecodingCodecs || []).indexOf(c.getAttribute("data-codec"))
-        }), page.querySelector("#chkHardwareEncoding").checked = config.EnableHardwareEncoding, $("#selectVideoDecoder", page).val(config.HardwareAccelerationType), $("#selectThreadCount", page).val(config.EncodingThreadCount), $("#txtDownMixAudioBoost", page).val(config.DownMixAudioBoost), page.querySelector(".txtEncoderPath").value = config.EncoderAppPath || "", $("#txtTranscodingTempPath", page).val(config.TranscodingTempPath || ""), $("#txtVaapiDevice", page).val(config.VaapiDevice || ""), page.querySelector("#selectH264Preset").value = config.H264Preset || "", page.querySelector("#txtH264Crf").value = config.H264Crf || "", page.querySelector("#chkEnableSubtitleExtraction").checked = config.EnableSubtitleExtraction || !1, page.querySelector("#selectVideoDecoder").dispatchEvent(new CustomEvent("change", {
+        }), page.querySelector("#chkHardwareEncoding").checked = config.EnableHardwareEncoding, $("#selectVideoDecoder", page).val(config.HardwareAccelerationType), $("#selectThreadCount", page).val(config.EncodingThreadCount), $("#txtDownMixAudioBoost", page).val(config.DownMixAudioBoost), page.querySelector(".txtEncoderPath").value = config.EncoderAppPathDisplay || "", $("#txtTranscodingTempPath", page).val(config.TranscodingTempPath || ""), $("#txtVaapiDevice", page).val(config.VaapiDevice || ""), page.querySelector("#selectH264Preset").value = config.H264Preset || "", page.querySelector("#txtH264Crf").value = config.H264Crf || "", page.querySelector("#chkEnableSubtitleExtraction").checked = config.EnableSubtitleExtraction || !1, page.querySelector("#selectVideoDecoder").dispatchEvent(new CustomEvent("change", {
```

Comment:
can you de-uglify at least this line?.. hard to tell what changed...

Reply:
config.EncoderAppPath was changed to config.EncoderAppPathDisplay.

I'm no js expert so wouldn't know how to beautify this code..

Reply:
Viewing the change without auto line wrap might help?

Reply:
de-uglification should look something like this:
```suggestion
        });
        page.querySelector("#chkHardwareEncoding").checked = config.EnableHardwareEncoding;
        $("#selectVideoDecoder", page).val(config.HardwareAccelerationType);
        $("#selectThreadCount", page).val(config.EncodingThreadCount);
        $("#txtDownMixAudioBoost", page).val(config.DownMixAudioBoost);
        page.querySelector(".txtEncoderPath").value = config.EncoderAppPathDisplay || "";
        $("#txtTranscodingTempPath", page).val(config.TranscodingTempPath || "");
        $("#txtVaapiDevice", page).val(config.VaapiDevice || "");
        page.querySelector("#selectH264Preset").value = config.H264Preset || "";
        page.querySelector("#txtH264Crf").value = config.H264Crf || "";
        page.querySelector("#chkEnableSubtitleExtraction").checked = config.EnableSubtitleExtraction || false;
        page.querySelector("#selectVideoDecoder").dispatchEvent(new CustomEvent("change", {
```